sources/packages/forms/src/form-definitions/studentprofile.json

```diff
@@ -1502,20 +1502,20 @@
           "data": {
             "values": [
               {
-                "label": "Male",
-                "value": "male"
+                "label": "Man",
+                "value": "man"
               },
               {
-                "label": "Female",
-                "value": "female"
+                "label": "Non-Binary",
+                "value": "nonBinary"
               },
               {
-                "label": "Unknown",
-                "value": "unknown"
+                "label": "Woman",
+                "value": "woman"
               },
               {
-                "value": "diverse",
-                "label": "Diverse"
+                "label": "Prefer not to answer",
+                "value": "preferNA"
```

Comment:
The extended version is preferred, for instance, `preferNotToAnswer`.
We can extend the DB column from varchar(10) to varchar(50).
Please let me know if you please that it would not be part of the ticket, we can talk with the team.

Reply:
Updated